File: Python/Ex_27.py
number = float(input("Enter the value you want to calculate the times table: "))

while(number <= 0):
    print("Only accpet positive numbers!")
    number = int(input("Enter the value you want to calculate the times table: "))

lower = float(input("Type the lower limit: "))
higher = float(input("Type the higher limit: "))

while(higher < lower):
    print("Invalid value")
    a = float(input("Type a value higher than the lower limit: "))

while(lower <= higher):
    result = number * higher
    print(f'{number} X {higher} = {result}')
    higher = higher - 1

File: c/Ex_27.c
#include <stdio.h>
#include <stdlib.h>

int main(int argc, char const *argv[])
{
    float result, number, lower, higher;

        printf ("Enter the value you want to calculate the times table: ");
        scanf ("%f", &number);

        while (number < 0)
        {
            printf ("Only accpet positive numbers! Enter the value you want to calculate the times table: ");
            scanf ("%f", &number);
        }

        printf ("Type the lower limit: ");
        scanf ("%f", & lower);
        
        printf ("Type the higher limit: ");
        scanf ("%f", & higher);

        while (lower >= higher)
        {
            printf ("Invalid value, type a value higher than the lower limit: ");
            scanf ("%f", & higher);
        }

        for (size_t; higher >= lower; higher--)
        {  
        result = higher * number;
        printf ("\n %.0f x %.0f = %.0f", number, higher, result);
        }
}